crypto/rand/drbg_lib.c

```diff
@@ -621,11 +618,8 @@ int RAND_DRBG_generate(RAND_DRBG *drbg, unsigned char *out, size_t outlen,
             || now - drbg->reseed_time >= drbg->reseed_time_interval)
             reseed_required = 1;
     }
-    if (drbg->parent != NULL) {
-        unsigned int reseed_counter = tsan_load(&drbg->reseed_prop_counter);
-        if (reseed_counter > 0
-                && tsan_load(&drbg->parent->reseed_prop_counter)
-                   != reseed_counter)
+    if (drbg->reseed_counter > 0 && drbg->parent != NULL) {
+        if (drbg->reseed_counter != drbg->parent->reseed_counter)
```

Comment:
Should it be `tsan_load(&drbg->parent->reseed_counter)` here?

I think the most likely consequence would be reseeding more rather than less often which is harmless.  I think it would be a very contrived case to miss a reseed.

Reply:
Yes, that's a mistake,  I'll fix it. Unfortunately, I can't simply add a fixup for d502cfadc974b7a107ea66c57c3db79544f15ebe, because it would not commute with the renaming in 3846caf23d0ea2435bdeeda5e2469f88e9aca5e2.  Instead I'm going to force-push the fixed commits without moving the merge-base, to give you a simple force-push diff to review.

Reply:
Fixed.